calculator.py

```
@@ -1,15 +1,31 @@
         return a - b
     def multiply(self,a,b):
         return a * b
     def divide(self,a,b):
         return a / b
-    def square_root(self,x):
-        return math.sqrt(x)
     def power(self,a,b):
-        return a ** b    
\ No newline at end of file
+        return a ** b    
+    def sqrt(self,n,guess=None):
```

Ruff

Missing return type annotation for public function `sqrt`

(ANN201)